src/lib/constants.js

```diff
@@ -107,12 +107,14 @@ export const TRANSFORMS_LIST = ['filter', 'groupby', 'aggregate', 'sort'];
 
 export const TRANSFORMABLE_TRACES = [
   'scatter',
-  'bar',
   'scattergl',
-  'histogram',
-  'histogram2d',
   'box',
   'violin',
+  'bar',
+  'ohlc',
```

Comment:
so here we've just added ohlc and candlestick, right?

Reply:
yes, that's all that can be done safely, box and violin are already there